### ISSUE TITLE
Provide a way to list packages from specific repositories

### ISSUE THREAD

**Issue by birdie-github** [rpm-software-management/dnf5#1837]
Which makes searching/grepping this list nearly impossible.

Using `dnf5-5.2.6.2-1.fc41.x86_64`.

Now it lists:
```
tmpwatch.x86_64                                 2.11-27.fc41                    fedora
tmt.noarch                                      1.38.0-1.fc41                   updates
```

It used to be:
```
tmpwatch.x86_64                                 2.11-27.fc41                    @fedora
tmt.noarch                                      1.38.0-1.fc41                   @updates
```

And now that I know how dnf list installed works, it should be `dnf list --installed`, there's an inconsistency:

```
xfconf.x86_64                                  4.18.3-3.fc41                      fedora
xfd.x86_64                                     1.1.2-2.fc34                       @commandline
```

So actually the AT symbol is still there for some "repos".

**Comment by pkratoch** [comment]
I think the main problem is that there is currently no good way to list packages from a specific repository. Grepping using the '@' symbol is just a workaround, but an option that would filter the list would be better.

Btw. the command repository-packages that could be used to achieve that as well is not yet implemented: https://github.com/rpm-software-management/dnf5/issues/915

**Comment by pkratoch** [comment]
I renamed the issue to better describe the use case rather than a specific implementation. Previously, it was "dnf list now omits the @ symbol for repos' names"

**Comment by birdie-github** [comment]
Packages names never include the at symbol, so you could just reinstate it, please.

**Comment by pkratoch** [comment]
> Packages names never include the at symbol, so you could just reinstate it, please.

I agree that package names never include the '@' symbol. I am only saying it's good to differentiate between what the use case is (in this case to find packages from a specific repository) and how to achieve the use case. There are multiple options of how to implement that and I would argue that adding the '@' symbol is not the best one. In any case, naming the issue based on the use case instead of a specific implementation seems better.

**Comment by birdie-github** [comment]
> Instead of grepping for @, you can grep for the third column.

This is how I've been doing that already, thanks but the command syntax is just a bit more complicated than using the AT symbol.
